src/components/implementation/no_interface/vkernel/vk_api.h

```diff
@@ -4,10 +4,10 @@
 #include "vk_types.h"
 
 /* extern functions */
-extern void vm_exit(void *);
 extern void vm_init(void *);
 extern void dom0_io_fn(void *);
 extern void vm_io_fn(void *);
+extern void *__inv_vkernel_serverfn(int a, int b, int c);
```

Comment:
you're right, it is supposed to be a VM management library. I'll move these externs to other appropriate place and take care of `__inv..` function too.